Cargo.toml

```diff
@@ -6,20 +6,24 @@ edition = "2021"
 
 [dependencies]
 adw = { git = "https://gitlab.gnome.org/World/Rust/libadwaita-rs", branch = "v0.3-adaptive-states-and-more", package = "libadwaita", features = ["v1_3"] }
-anyhow = "1.0"
+anyhow = "1"
 ellipse = "0.2"
 futures = { version = "0.3", default-features = false }
 gettext-rs = { version = "0.7", features = ["gettext-system"] }
-gtk = { version = "0.6", package = "gtk4", features = ["gnome_43", "v4_10", "blueprint"] }
+gtk = { version = "0.6", package = "gtk4", features = ["gnome_44", "blueprint"] }
 image = { version = "0.24", default-features = false, features = ["jpeg", "png", "webp"] }
-indexmap = "1.9"
+indexmap = "1"
 locale_config = "0.3"
 log = "0.4"
-once_cell = "1.17"
-pretty_env_logger = "0.4"
-qrcode-generator = { version = "4.1", default-features = false }
-regex = "1.8"
+once_cell = "1"
+pretty_env_logger = "0.5"
+qrcode-generator = { version = "4", default-features = false }
+regex = "1"
 rlt = { package = "gtk-rlottie", git = "https://github.com/YuraIz/gtk-rlottie-rs" }
```

Comment:
I think I'll publish that crate later

Reply:
Ok, I'll wait till then before merging. Also, I forgot to include flathub beta instructions in the readme.

Reply:
I think it would be better to update rlt first, so I'll make a separate PR for that

Reply:
Ok, I just updated the Flathub Beta Instructons, if you want to have a look.

Reply:
I like the updated instructions